src/components/sections-display.js

```diff
@@ -27,26 +27,11 @@ function SectionsDisplay({
   onSectionPasteFromClipboard,
   onSectionHardDelete
 }) {
-  const [isDragging, setIsDragging] = useState(false);
+  const droppableIdRef = useRef(useId());
```

Comment:
:partying_face: first usage, yay!

Reply:
Yeees! 😁 I'm happy I could remember about it 😅 Pretty handy!